.changeset/beige-pens-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+"@accelint/design-toolkit": patch
```

Comment:
does the update from `unclass` > `unclassified` make this a "major" version bump per Brenna's comment about that being a breaking change?